projects/FF15IPQEXT.yaml

```diff
@@ -0,0 +1,10 @@
+Department: Chemistry
+Description: Parameterizing force fields for artificial amino acids through molecular dynamic simulations 
+FieldOfScience: Chemistry
+ID: '611'
+Name: FF15IPQEXT
```

Comment:
I thought we were using the `<Institution>_<PI Last name>` format?

Reply:
We are. This is a project that was created before this standardization was implemented but wasn't added to topology.